demo/addons/fmod/FmodPlugin.gd

```diff
@@ -1,5 +1,11 @@
 @tool
 extends EditorPlugin
 
-func _init():
-	pass
+const ADDON_PATH = "res://addons/fmod"
+
+@onready var iconFmodEventEmitter:Texture = preload(ADDON_PATH + "/icons/FmodEventEmitter2D.svg")
+@onready var theme = get_editor_interface().get_base_control().get_theme()
+
+func _ready() -> void:
+	# configure icons
+	theme.set_icon("FmodEventEmitter2D", "EditorIcons", iconFmodEventEmitter)
```

Comment:
There is no way to currently set editor icons within GDExtension. Until https://github.com/godotengine/godot-cpp/issues/863 is implemented we will have to use this workaround.